src/cancelContentPlay.js

```diff
@@ -32,11 +32,21 @@ const cancelContentPlay = function(player) {
     player.el_.style.display = 'none';
 
     // Unhide the player and remove the placeholder once we're ready to move on.
-    player.one(['adplaying', 'adtimeout', 'adserror', 'adscanceled', 'adskip',
+    player.one(['adstart', 'adplaying', 'adtimeout', 'adserror', 'adscanceled', 'adskip',
```

Comment:
I was originally trying to hide the black div using fullscreenschange but in one scenario (I think it was with a specific ad plugin) fullscreenchange wasn't firing and hiding the div.  Additionally, it wasn't being removed in the player.one('adplaying') case so I used adstart. The risk is removing the div too soon and allowing content flash on iOS without playsinline.